src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -117,53 +121,44 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter, I
                 if (allowServeFromCache && cacheItem != null) {
 
                     Logger.Debug("Item '{0}' was found in cache.", _cacheKey);
+                    
+                    if(cacheItem.IsValid(_now)) {
```

Comment:
I don't know what I changed, all my `if` are not formatted automatically anymore :/